rfcs/0085-nixos-release-stablization.md

```diff
@@ -0,0 +1,131 @@
+---
+feature: nixos-release-stablization
+start-date: 2021-01-17
+author: Jonathan Ringer (@jonringer)
+co-authors:
+shepherd-team: @ryantm, @garbas, @Mic92
+shepherd-leader: @ryantm
+related-issues: [NixOS release schedule](https://github.com/NixOS/rfcs/pull/80)
+---
+
+# Summary
+[summary]: #summary
+
+To bring more certainty to the release cycle, add short periods where
+breaking changes are partially restricted. A list of Release Critical
+Packages is defined. Also, move most release stablization work from
+the `release` branch to the `master` branch to reduce backports.
+
+# Motivation
+[motivation]: #motivation
+
+We want to release on time without herculian efforts. Limited
+restrictions to merging breaking changes to `staging` should avoid the common
+practice of forcing many, largely-untested, commits into master before
+branch-off.
+
+This happened for the 20.09 release in which ~640 staging
+commits were pushed through in the days leading up to the branch off
+([#95492](https://github.com/NixOS/nixpkgs/pull/95492),
+[#96280](https://github.com/NixOS/nixpkgs/pull/96280),
+[#96437](https://github.com/NixOS/nixpkgs/pull/96437),
+[#97146](https://github.com/NixOS/nixpkgs/pull/97146)). The last PR included a
+release blocker fix (introduced in [one of the mentioned
+PRs](https://github.com/NixOS/nixpkgs/pull/96437)) which delayed the
+branch-off from 4 Sept 2020 to 7 Sept 2020, and ZHF had to be delayed
+another day to get a semi-accurate hydra build status. The systemd
+bump was also largely responsible delaying the release until after the
+scheduled release date where basic login features in plasma [weren't
+restored until Oct 5th, a week after the planned release
+date](https://github.com/NixOS/nixpkgs/pull/99629) and a backlog of
+other plasma related issues [weren't resolved until Oct 19th, almost a
+month after the planned release
+date](https://github.com/NixOS/nixpkgs/pull/101078).  This is not to
+discredit any individual, but demonstrate that certain packages have
+many dimensions of compatibility which need a longer time to stabilize
+on unstable before being included in a release.
+
+
+# Definitions
+[definitions]: #definition
+- Breaking change
+  - A change that is likely to break downstream.
+  - For projects following Semantic Versioning, major version bumps indicate breaking changes.
+- Restricted
+  - Breaking changes are disallowed.
+  - Breaking changes for security or critical bugs are allowed. (Patching is preferred over updates.)
+  - New packages and NixOS modules are welcome.
+- Unrestricted
+  - Breaking changes are allowed.
+  - New packages and NixOS modules are welcome.
+- ZHF: Zero Hydra Failures
+  - Period in which packages are stabilized in preparation of a release.
+
+# Changes
+
+## Release Critical Packages
+
+A list of Release Critical Packages will be maintained in the 
+[release wiki](https://github.com/NixOS/release-wiki).
```

Comment:
```suggestion
[release wiki](https://nixos.github.io/release-wiki/).
```

Reply:
Didn't realize this was a thing, awesome.